new.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>SlimStampen Learning App</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+		<script type="text/javascript">var fs = require("fs");</script>
+		<script data-main="app/formpage" src="lib/require.js"></script>
+    <link rel="stylesheet" href="./resources/css/bootstrap.css">
+    <link rel="stylesheet" href="./resources/css/custom.css">
+    <link rel="stylesheet" href="./node_modules/parsleyjs/src/parsley.css">
+    <link rel="stylesheet" href="./node_modules/bootstrap-select/dist/css/bootstrap-select.min.css">
+    <!-- Javascript libraries are added before the </body> tag  -->
+  </head>
+	<body>
+    <nav class="navbar navbar-inverse navbar-fixed-top">
+     <div class="navbar-header" >
+       <ul class="nav navbar-nav " id="nav-left">
+         <li><a class="navbar-brand" href="#" onclick="location.href='index.html';" >SlimStampen</a></li>
+       </ul>
+
+       <div class="btn-group center-menu" role="group">
+         <button type="button" class="btn btn-default">
+           <span class="glyphicon glyphicon-file" aria-hidden="true"></span> New
+         </button>
+         <button type="button" class="btn btn-default">
+           <span class="glyphicon glyphicon-folder-open" aria-hidden="true"></span>&nbsp; Open
+         </button>
+         <button type="button" class="btn btn-default">
+           <span class="glyphicon glyphicon-save" aria-hidden="true"></span> Download
+         </button>
+         <button type="button" class="btn btn-default" onclick="location.href='upload.html';">
+           <span class="glyphicon glyphicon-open" aria-hidden="true"  ></span> Upload
+         </button>
+       </div>
+
+       <div class="btn-group right-menu" role="group">
+         <button type="button" class="btn btn-default">
+           <span class="glyphicon glyphicon-user" aria-hidden="true"></span> Hedderik
+         </button>
+         <button type="button" class="btn btn-default">
+           <span class="glyphicon glyphicon-cog" aria-hidden="true"></span>
+         </button>
+         <button type="button" class="btn  btn-danger dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+           <span class="glyphicon glyphicon-off"></span> <span class="caret"></span>
+         </button>
+         <ul class="dropdown-menu dropdown-menu-right">
+           <li><a href="#">Pause lesson</a></li>
+           <li><a href="#">Quit lesson</a></li>
+           <li role="separator" class="divider"></li>
+           <li><a href="#">Help</a></li>
+         </ul>
+       </div>
+     </div>
+   </nav>
+		<div id="content">
```

Comment:
Strange formatting. You could use a hook with an auto formatter, or simply use an auto formatter every now and then.